prisma/migrations/0_init/migration.sql

```diff
@@ -3,7 +3,9 @@ CREATE TABLE "Site" (
     "id" TEXT NOT NULL,
     "name" TEXT NOT NULL,
     "type" TEXT NOT NULL,
-    "location" TEXT NOT NULL,
+    "position" TEXT NOT NULL,
+    "positionInfo" TEXT NOT NULL,
```

Comment:
If you want positionInfo to always contain the city and the country that the site is in, I would rather that you just store that as two separate fields here to make it clear and easier to use